css/design_system/utils/text.scss

```
@@ -0,0 +1,82 @@
+@mixin docstring($class, $prop) {
+  /**
+   * @title #{$class}
+   * @category utils
+   * @util_type #{$prop}
+   *
+   * @example
+   * <p class="#{$class}">Example text</p>
+   */
+}
+
+@mixin docstring_messy($class, $prop) {
+  /**
+   * @title #{$class}
+   * @category utils
+   * @util_type #{$prop}
+   *
+   * @example
+   * <p class="#{$class}">exaMple tEXT</p>
+   */
+}
+
+// text-align
+.s-text-start {
+  @include docstring("s-text-start", "text-align");
+  text-align: start;
+}
```

I think we've been putting the docs outside of the class definition, e.g.:
```suggestion
@include docstring("s-text-start", "text-align");
.s-text-start {
  text-align: start;
}
```